### PR TITLE
Improved PR review prompt to generate more concise review comments

### PR DESCRIPTION
**Pull request by eugene-yaroslavtsev**
How I did it: https://sharegpt.com/c/tWFOJoa

- I’m so lazy that I even use chatGPT to improve a chatGPT prompt
- one notable thing is that I combined what I generated with a newer version of the prompt that's on the original repo I forked  because we had MM repo pinned to a specific version and were using an older prompt
- ^ only real change I added though was the bullet-point list from newer version
  - I particularly liked last item they added: "- Best practices: DRY, SOLID, KISS"

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -164,40 +161,42 @@ inputs:
...
+      "Test", "Chore", "Revert", and provide a bullet point list. For example:
+      "New Feature: An integrations page was added to the UI". Keep your
+      response within 50-100 words. Avoid additional commentary as this response
       will be used as is in our release notes.
 
-      Below the release notes, generate a short, celebratory poem about the 
-      changes in this PR and add this poem as a quote (> symbol). You can 
+      Below the release notes, generate a short, celebratory poem about the
+      changes in this PR and add this poem as a quote (> symbol). You can
       use emojis in the poem, where they are relevant.
```
Comment (eugene-yaroslavtsev) [inline action.yml:200]: Just auto formatting silliness ignore this

File: action.yml [hunk 2]
```diff
@@ -148,10 +148,7 @@ inputs:
     required: false
     description: 'System message to be sent to OpenAI'
     default: |
-      You are `@coderabbitai` (aka `github-actions[bot]`), a language model 
-      trained by OpenAI. Your purpose is to act as a highly experienced 
-      software engineer and provide a thorough review of the code hunks
-      and suggest code snippets to improve key areas such as:
+            You are `@openai` (aka `github-actions[bot]`), a language model trained by OpenAI. Act as a seasoned software engineer to review the code hunks. Focus on key areas such as:
```
Comment (mm-zacharydavison) [inline action.yml]: Does this break the `@coderabbitai` tagging functionality?

File: action.yml [hunk 3]
```diff
@@ -164,40 +161,42 @@ inputs:
...
         - Best practices: DRY, SOLID, KISS
 
-      Do not comment on minor code style issues, missing 
-      comments/documentation. Identify and resolve significant 
-      concerns to improve overall code quality while deliberately 
-      disregarding minor issues.
+      Provide concise initial feedback. If more context or explanation is requested, then elaborate further. Avoid commenting on minor style issues or giving compliments unless asked. If unfamiliar APIs or methods appear, assume they are current.
+
+      Example interaction:
+      Reviewer: "You forgot an import."
```
Comment (mm-zacharydavison) [inline action.yml]: Surely we'd want it to specify which import at least? We just don't want the preamble.

**Review by mm-zacharydavison — CHANGES_REQUESTED** [review]
Could you test this by rebasing some PR onto this branch and asking it to review it?

File: action.yml (hunk 2 above)
Comment (eugene-yaroslavtsev) [reply action.yml]: Good point: https://github.com/coderabbitai/ai-pr-reviewer/blob/main/src/review-comment.ts#L20

Changing it back to `@coderabbitai` for now.

**Comment by mm-zacharydavison** [comment]
> Could you test this by rebasing some PR onto this branch and asking it to review it?

Did you test this anywhere? Can you show an example?

**Review by mm-zacharydavison — CHANGES_REQUESTED** [review]
Just waiting to see it tested on a PR.

**edit:** nevermind, I noticed this isn't in monorepo. Approved.

**Review by mm-zacharydavison — APPROVED** [review]

**Pull request merged**